File: CSharp/DependencyBreakingKatas/D/RestCountries/CountryDescription.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Org.Codecop.Dependencies.D.Restcountries
{
    public class CountryDescription
    {
        public Codes codes { get; set; }

        public string region { get; set; }

        public Coordinates coordinates { get; set; }

        [JsonExtensionData]
        private IDictionary<string, object> additionalProperties = new Dictionary<string, object>();

    }
}

File: Java/src/main/java/org/codecop/dependencies/d/restcountries/CountryDescription.java
package org.codecop.dependencies.d.restcountries;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
public class CountryDescription {

    public Codes codes;

    public String region;

    public Coordinates coordinates;

    public Codes getCodes() {
        return codes;
    }

    public String getRegion() {
        return region;
    }

    public Coordinates getCoordinates() {
        return coordinates;
    }

}
